internal/command/arguments/extended.go

```diff
@@ -85,20 +89,20 @@ type Operation struct {
 	// method Parse to populate the exported fields from these, validating
 	// the raw values in the process.
 	targetsRaw      []string
+	excludesRaw     []string
 	forceReplaceRaw []string
 	destroyRaw      bool
 	refreshOnlyRaw  bool
 }
 
-// Parse must be called on Operation after initial flag parse. This processes
-// the raw target flags into addrs.Targetable values, returning diagnostics if
-// invalid.
-func (o *Operation) Parse() tfdiags.Diagnostics {
+// parseTargetables gets a list of strings, each representing a targetable object, and returns a list of
+// addrs.Targetable
+// This is used for parsing the input of -target and -exclude flags
+func parseTargetables(rawTargetables []string) ([]addrs.Targetable, tfdiags.Diagnostics) {
+	var targetables []addrs.Targetable
 	var diags tfdiags.Diagnostics
 
-	o.Targets = nil
-
-	for _, tr := range o.targetsRaw {
+	for _, tr := range rawTargetables {
```

Comment:
the error message here still says "Invalid Target" but it could be exclude.

Reply:
2076f53523e55944bcc4a18871125a5890f2c502